operator-metadata/image.yaml

```diff
@@ -23,13 +23,13 @@ labels:
   - name: "operators.operatorframework.io.bundle.package.v1"
     value: "amq-streams"
   - name: "operators.operatorframework.io.bundle.channels.v1"
-    value: "stable,amq-streams-2.x,amq-streams-2.2.x"
+    value: "stable,amq-streams-2.x,amq-streams-2.3.x"
   - name: "operators.operatorframework.io.bundle.channel.default.v1"
     value: "stable"
   - name: "com.redhat.delivery.operator.bundle"
     value: "true"
   - name: "com.redhat.openshift.versions"
-    value: "v4.6"
+    value: "v4.9"
```

Comment:
Does this refer to the oldest supported version?

Reply:
It refers to the oldest OCP version which the Streams bundle will be shipped to